convert/lib/convert.py

```diff
@@ -634,19 +634,28 @@ def _get_nested_stacks(self, stack_name: str) -> list:
     def delete_stack(self):
         self.setup()
 
-        cf_tf_output = json.loads(
-            run(["terraform", "output", "-json"], cwd="cloudformation-only", capture_output=True).stdout
-        )
-        if not (cf_only_role := cf_tf_output.get("cloudformation_only_role")) or not re.match(
-            "arn:.*:iam:.*role/cloudformation-only", cf_only_role["value"]
-        ):
+        cf_only_state_file = os.path.join(os.getcwd(), "cloudformation-only", "terraform.tfstate")
```

Comment:
Dunno if there was a problem that was hit, but it seems more proper to get data out of terraform via the output than to dig into its data files (not that I really expect it to matter much either way).

Reply:
There is something weird happening with the encoding on terraform 1.5 thats breaking the json_load. I think querying the state is safer and more deterministic

Reply:
If it solves a problem, that's cool. Philosophically, the output should be an API that terraform can use between it and external things to ensure that data interchange format is predictable, as where we shouldn't assume the statefile will have any particular format. Pragmatically, it sounds from what you're saying that the opposite is true today, sooo... :)